todo_app/ViewModel.py

```diff
@@ -0,0 +1,146 @@
+from datetime import datetime, date, timedelta
+
+class ViewModel:
+
+    def __init__(self, items, todo, doing, done, limitdone= True):
+        self._items = items
+        self._todo = todo
+        self._doing = doing
+        self._done = done
+        self._limitdone = limitdone
+        #self._displaylimit = displaylimit
+        
+    #returned cards from API call to Trello
+    @property
+    def items(self):
+        
+        return self._items
+
+    #Key values for columns
+    @property
+    def todo(self):
+        return self._todo
+
+    @property
+    def doing(self):
+        return self._doing
+
+    @property
+    def done(self):
+        return self._done
+
+    @property
+    def limitdone(self):
```

Comment:
I don't think these properties are ever used. You can delete them (but keeping the lines setting e.g. self._limitdone inside the init function)

Reply:
removed